.github/workflows/examples.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         # NOTE: add new example projects here. The example project has to
         # satisfy some requirements for this workflow to work correctly. See
         # ./examples/README.md (with respect to the repository root).
-        example: ['botan', 'c-minisat', 'libpcap', 'c-qrcode']
```

Comment:
I can seen an error in CI because the example is named `blocktest` here so it expects an `examples/blocktest` directory to exist, but the directory is called `examples/blocks` instead